src/main/scala/io/conduktor/ksm/source/HttpSourceAcl.scala

```diff
@@ -0,0 +1,198 @@
+package io.conduktor.ksm.source
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.google.auth.oauth2.{GoogleCredentials, IdTokenCredentials, IdTokenProvider, ServiceAccountCredentials}
+import com.typesafe.config.Config
+import io.conduktor.ksm.parser.AclParserRegistry
+import org.slf4j.LoggerFactory
+import skinny.http.{HTTP, HTTPException, Request, Response}
+
+import java.io.{StringReader, _}
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+import java.util.Collections
+
+class HttpSourceAcl(parserRegistry: AclParserRegistry)
+  extends SourceAcl(parserRegistry) {
+
+  private val log = LoggerFactory.getLogger(classOf[HttpSourceAcl])
+
+  private val jwtMap = collection.mutable.Map[String, String]()
+  private val expirationMap = collection.mutable.Map[String, Instant]()
```

Comment:
why these maps ? the key is the audience but it seems the audience never change

Reply:
You're no really needed